ballerina-shell/modules/shell-cli/src/main/java/io/ballerina/shell/cli/BShellConfiguration.java

```diff
@@ -159,14 +168,26 @@ public Builder setDumb(boolean dumb) {
             return this;
         }
 
+        /**
+         * Start file is the file containing declarations that should
+         * be loaded into the shell initially.
+         * If the file loading fails, the shell will not start and fail.
+         * The file should not contain any definitions for built-in names.
+         * (eg: main, run, init)
```

Comment:
Why do we need this restriction? Is it due to how we generate code?

Reply:
yeah, they are already kinda defined in the shell, so user should not be able to redefine them